tinychat/models/llama.py

```diff
@@ -259,9 +265,10 @@ def forward(
         start_pos: int,
         freqs_cis: torch.Tensor,
         mask: Optional[torch.Tensor],
+        chunk_prefill:bool,
     ):
         h = x + self.self_attn.forward(
-            self.input_layernorm(x), start_pos, freqs_cis, mask
+            self.input_layernorm(x), start_pos, freqs_cis, mask,chunk_prefill
```

Comment:
Please leave a blank space between comma and `chunk_prefill`. Same for the rest of the modifications.